test/int/project_test.go

```diff
@@ -209,7 +209,7 @@ var _ = Describe("AtlasProject", Label("int", "AtlasProject"), func() {
 			createdProject.ObjectMeta = expectedProject.ObjectMeta
 			Expect(k8sClient.Create(context.Background(), expectedProject)).ToNot(HaveOccurred())
 
-			expectedCondition := status.FalseCondition(status.ProjectReadyType).WithReason(string(workflow.AtlasCredentialsNotProvided))
+			expectedCondition := status.FalseCondition(status.ProjectReadyType).WithReason(string(workflow.Internal))
```

Comment:
Why is this error type now less specific?

Reply:
I replaced the reason for a more specific instead of using the generic one